baseplate/config.py

```diff
@@ -167,6 +167,24 @@ def Base64(text):
         raise ValueError(*exc.args)
 
 
+def File(mode="r"):
+    """A path to a file.
+
+    This takes a path to a file and returns an open file object, like
```

Comment:
Hmm, so this doesn't actually return an open file object, it returns a function that returns an open file object, right?  So they'd still need to do something like `with config['foo']() as f:`?  (This is more a note on the docstring and maybe making the usage a bit clearer).

I guess I'm wondering if this really provides enough functionality to justify the abstraction.  In other words: why not just let the author of the application take the path in the config and `open` it and avoid the potential confusion of whether or not the file needs to be closed?